modules/microshift-nw-config-route-admission.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-route-admission_{context}"]
+= Configuring route admission policy
+
+By default, {microshift-short} prevents routes in multiple namespaces from using the same hostname. To accommodate different use cases, you can use the configuration option below to disable the `namespaceOwnership` check. This allows routes to claim different paths of the same hostname across namespaces.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. Update the {microshift-short} `config.yaml` file parameter in the `ingress.routeAdmissionPolicy.namespaceOwnership` field to `InterNamespaceAllowed` as shown in the following example to disable the check:
++
+.Example `config.yaml` route admission policy settings
```

Comment:
```suggestion
.Example `config.yaml` route admission policy
```
We're setting a single policy, so I think it's OK to shorten for the sake of minimalism.

Reply:
done